src/app/cli/src/coda.ml

```diff
@@ -119,6 +125,30 @@ let daemon logger =
        let compute_conf_dir home =
          Option.value ~default:(home ^/ ".coda-config") conf_dir
        in
+       let%bind log_level =
+         match log_level with
+         | None ->
+             Deferred.return Logger.Level.Warn
+         | Some log_level_str -> (
+             let open Logger in
+             match Level.of_string log_level_str with
```

Comment:
can you normalize the case on `log_level_str`?

Reply:
Level.of_string seems to already do this, I tried with "warn" and "Warn" and both seemed to work.

Reply:
@bkase actually I tried again with stuff like WARN and wArN and those didn't get accepted so I added a lowercase on everything here